hw03_NatsarS.py

```diff
@@ -0,0 +1,68 @@
+#################################################################################
+# Author: Salaam Natsar
+# Username: NatsarS
+#
+# Assignment: Homework 03
+# Purpose: Created a sun flower
+# Google Doc Link: https://docs.google.com/document/d/1gRRxdAQvTMhQ6pwa0BwnBMu09rZ2cglwKLnd5j2zvpg/edit?usp=sharing
+#################################################################################
+# Acknowledgements:
+# https://stackoverflow.com/questions/32804572/how-to-hide-a-turtle-icon-pointer-in-python
+#
+#################################################################################
+import turtle
+
+def main():
```

Comment:
Missing a docstring (main's docstring should usually state the purpose of the entire program).